mailscanner/msre_edit.php

```diff
@@ -103,89 +81,77 @@
     if (isset($_POST['submitted'])) {
         if (false === checkFormToken('/msre_edit.php form token', $_POST['formtoken'])) {
             die(__('dietoken99'));
         }
 
         list($bytes_written, $status_message) = Process_Form($file_contents, $short_filename);
-        // re-read the file after processing
+        // Re-read the file after processing
         $file_contents = Read_File($full_filename, $bytes_written);
     }
 
-    // the form always gets displayed, even if it was submitted, so
+    // The form always gets displayed, even if it was submitted, so
     // display the form now
     Show_Form($status_message, $short_filename, $file_contents, $CONF_ruleset_keyword);
-    // clear status message
+    
+    // Clear status message
     $status_message = '';
-
-    echo "</table><tr><td>\n";
+    echo '</table><tr><td>' . "\n";
     html_end();
 }
 
-// #################
-// ### Functions ###
-// #################
-
+// Functions
 
+/**
+ * @param string status_msg
+ * @return displays the form
+ */
 function Show_Form($status_msg, $short_filename, $file_contents, $CONF_ruleset_keyword)
 {
-    // displays the form
-    //
-    // inputs:
-    // 		status_msg     		if there's anything in the status_msg variable,
-    // 						it'll be printed below the header when the form
-    // 						is displayed
-    //
-    // ouput:
-    // 		displays the form
-    //
-
-    // display top of page stuff
-    echo "<table border=\"0\" class=\"mailwatch\" align=\"center\">\n";
-    echo '<form method="post" name="MSRE_edit" action="msre_edit.php">' . "\n";
-    echo '<INPUT TYPE="HIDDEN" NAME="file" VALUE="' . $short_filename . '">' . "\n";
-    echo '<INPUT TYPE="HIDDEN" NAME="token" VALUE="' . $_SESSION['token'] . '">' . "\n";
-    echo '<INPUT TYPE="HIDDEN" NAME="formtoken" VALUE="' . generateFormToken('/msre_edit.php form token') . '">' . "\n";
-    echo '<input type="SUBMIT" name="submit" value="' . __('submit04') . '">' . "\n";
-
-    echo "<input type=\"hidden\" name=\"submitted\" value=\"1\">\n";
-    // check for status message, and append it to the end of the header
-
+    // Display top of page stuff
+    echo '<table border="0" class="mailwatch" align="center">' . "\n";
+    echo '<form method="post" name="msre_edit" action="msre_edit.php">' . "\n";
+    echo '<input type="hidden" name="file" value="' . $short_filename . '">' . "\n";
+    echo '<input type="hidden" name="token" value="' . $_session['token'] . '">' . "\n";
+    echo '<input type="hidden" name="formtoken" value="' . generateformtoken('/msre_edit.php form token') . '">' . "\n";
+    echo '<input type="submit" name="submit" value="' . __('submit55') . '">' . "\n";
+    echo '<input type="hidden" name="submitted" value="1">' . "\n";
+    
+    // Check for status message, and append it to the end of the header
     $my_header = '';
     if ($status_msg) {
-        $my_header .= "<br>\n" . $status_msg;
+        $my_header .= '<br>\n' . $status_msg;
```

Comment:
the \n will be printed as line break but as the plain chars `\` and `n`. Has to be in double quotes or use PHP_EOL